translations/translation_utility.sh

```diff
@@ -42,7 +42,11 @@ function Confirmation() { # Process Yes/No/Quit response
 # Functions End
 
 # ===== Modules ====
-Check_bin() {                             #C heck if the bin path exists
+Check_bin() { # Check if the bin path exists
+    bin_exists=$(which lupdate)
+    if [[ ! $bin_exists ]]; then
+        echo 123123321321
```

Comment:
Huh?

Reply:
This shouldn't be included in this stage.

Reply:
Something like this maybe?
```
if ! $(which lupdate 2>/dev/null); then
    echo lupdate/lrelease not in PATH
fi
```

Reply:
And in Ubuntu it will be like
`
/usr/lib/x86_64-linux-gnu/qt4/bin/qmake'
`

Reply:
I will use `which` for Linux sys.

Reply:
a) sure not "qt4" but "qt5" and b) ending before the "qmake"

Reply:
Better use `/usr/bin/env` than `which`, that'd be more platform and shell independant.

if $(/usr/bin/env lupdate 2>/dev/null)
then
    : found it
else
    echo lupdate/lrelease not in PATH
fi

My son's Linux machin has lupdate(lrelease in /usr/bin, so in the standard PATH

Reply:
I searched on the Internet. The results said that Linux systems￼ has built in￼￼ qt so I think we don't have to worry about that. And the code is almost done￼￼

Reply:
Yep, that's what I just found too.
Check whether available using `env`, if not available add the Mac default, like you did and check again, if still not available, ask for it